src/display/Plot/OfY.tsx

```diff
@@ -3,20 +3,24 @@ import { usePaneContext } from "../../context/PaneContext"
 import { Parametric, ParametricProps } from "./Parametric"
 import { vec } from "../../vec"
 
-export interface OfYProps extends Omit<ParametricProps, "xy" | "t"> {
+export interface OfYProps extends Omit<ParametricProps, "xy" | "domain" | "t"> {
   x: (y: number) => number
+  domain?: {min?: number, max?: number}
```

Comment:
So at the moment, `t` is a vec.Vector2 (array `[min, max]`) in Parametric—I think it will confuse people to mix the two different ways of declaring a domain.

I'd be open to discussing whether an object or a tuple here is better, but what matters to me is consistency—and I think until 1.0 we couldn't change away from a tuple since it's what Parametric uses.

Reply:
I like consistency, as well. I'm happy to change it to a tuple. I suppose that if a consumer wants to limit just one of the bounds, they can use +/-Infinity for the other value (just like what I placed in the code).